src/Microsoft.Health.Fhir.Stu3.Api/Microsoft.Health.Fhir.Stu3.Api.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Ensure.That" Version="10.1.0" />
     <PackageReference Include="FluentValidation" Version="9.5.4" />
     <PackageReference Include="Hl7.Fhir.STU3" Version="$(Hl7FhirVersion)" />
+    <PackageReference Include="IdentityServer4" Version="4.1.2" />
```

Comment:
>IdentityServer4 [](http://example.com/codeflow?start=31&length=15)

What we have this project for stu3 and not in others?

Reply:
I added while doing E2E, but it's not necessary. Removed!